crates/bevy_render/src/render_phase/mod.rs

```diff
@@ -84,59 +83,12 @@ impl<I: PhaseItem> RenderPhase<I> {
         let mut draw_functions = draw_functions.write();
         draw_functions.prepare(world);
 
-        for item in &self.items {
+        let mut index = 0;
+        while index < self.items.len() {
+            let item = &self.items[index];
             let draw_function = draw_functions.get_mut(item.draw_function()).unwrap();
             draw_function.draw(world, render_pass, view, item);
-        }
-    }
-
-    /// Renders all [`PhaseItem`]s in the provided `range` (based on their index in `self.items`) using their corresponding draw functions.
-    pub fn render_range<'w>(
```

Comment:
We can't remove render_range(). It's important to this PR https://github.com/bevyengine/bevy/pull/8015.